magda-web-client/src/UI/ChartConfig.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+//import PropTypes from 'prop-types';
```

Comment:
Do you not need this anymore? If so just delete it :)

Reply:
Previously was a placeholder.
@chloeleichen has done the chart section.
I will merge her PR into this branch to cover this later.